specs/signature-specification.md

```diff
@@ -51,6 +51,55 @@ Signature Manifest Example per OCI artifact manifest
   - **`io.cncf.notary.x509chain.thumbprint#S256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprint of signing certificate and certificate chain (including root) used for signature generation. The annotation name contains the hash algorithm as a suffix (`#S256`) and can be extended to support other hashing algorithms in future.
     The list of fingerprints is present as a JSON array string, corresponding to ordered certificates in [*Certificate Chain* unsigned attribute](#unsigned-attributes) in the signature envelope.
 
+### Backwards Compatibility
+
+Notary v2 MAY support using [OCI Image manifest][oci-image-manifest] to store the signature in the registries that implement partial or older versions of the OCI Image specification.
```

Comment:
This statement is not quite accurate. The `subject` field for the OCI image manifest is introduced in `image-spec v1.1`. Therefore, registries supporting OCI image-spec v1.0 may still not support OCI image manifest with the `subject` field.

Reply:
> This statement is not quite accurate. The `subject` field for the OCI image manifest is introduced in `image-spec v1.1`. Therefore, registries supporting OCI image-spec v1.0 may still not support OCI image manifest with the `subject` field.

when pushing the fallback manifest, oras library handles the complexities of creating the relationship between artifact and subject via fallback mechanism.

Reply:
@shizhMSFT - I think if registries don't support additional properties like the subject, I think its fair to say that the it doesn't support extended fields for even Image-spec 1.0 and maybe we should not handle that. For e.g. example the registry might not even support index. From the tool perspective do you think having the subject property in v1.0 is ok?

Reply:
From the tool perspective, having the `subject` property in `v1.0` is definitely ok since it is indeed partial `v1.1`.

My focus is the accuracy of the wording in the spec documentation. That is, implementing `image-spec v1.0` does not imply supporting the `subject` field.

How about
```suggestion
Notary v2 MAY support using [OCI image manifest][oci-image-manifest] to store the signature in the registries that implement partial OCI Image specification v1.1.
```

Reply:
@shizhMSFT How about I update the link of `oci-image-manifest` to `OCI image manifest v1.1.0-rc.2` to indicate that notation using OCI image manifest with `subject` field, while keep the wording `implement partial or older versions of the OCI Image specification. ` Because with your suggestion, it seems registries implementing v1.0 are excluded. I think some registries implementing v1.0 support `subject` field, since it is an optional field.

Reply:
> some registries implementing v1.0 support `subject` field

Yes, you may consider those registries partially v1.1 compatible.